### PR TITLE
MTM-56850 sso redirect to ui application documentation

### PR DESCRIPTION
**Pull request by mmed-c8y**

### CODE REVIEW

**Comment by kgol-c8y** [comment]
I think if you add the screenshot with the enabled `Redirect to the user interface application` toggle button and the option to add `redirect URL` is greyed out, so that it makes sense that user need not add redirect URL

File: content/authentication/sso-bundle/custom-template.md [hunk 1]
```diff
@@ -31,6 +31,7 @@ The **Basic** section of the **Single sign-on** page consists of the following c
 |Field|Description|
 |:---|:---|
 |Redirect URI|Redirect parameter. Can be used in request definitions as a ${redirectUri} placeholder
+|Redirect to the user interface application|The redirect URL is automatically set to the application used by the user during login.
```
Comment (kgol-c8y) [inline content/authentication/sso-bundle/custom-template.md:34]: please look into my comment in the conversation

**Review by kgol-c8y — CHANGES_REQUESTED** [review]

**Review by mbak-c8y — APPROVED** [review]

**Comment by kgol-c8y** [comment]
Closing this as its duplicate of https://github.com/SoftwareAG/c8y-docs/pull/2327

**Comment by kgol-c8y** [comment]
Closing this as its duplicate of https://github.com/SoftwareAG/c8y-docs/pull/2327

**Pull request closed without merging**